axlearn/audio/asr_decoder.py

```diff
@@ -196,19 +196,30 @@ def _compute_target_paddings(
         target_labels: Tensor = input_batch["target_labels"]
         # Infer target_paddings from out-of-range labels.
         target_paddings = jnp.logical_or(cfg.vocab_size <= target_labels, target_labels < 0)
+        return target_paddings
 
+    def _input_stats_summary(
```

Comment:
```suggestion
    def _input_stats_summaries(
```
or `def _add_input_stats_summaries` if we decide to inline the add, which may be more similar to other callsites in the repo.

Reply:
Have the changes been pushed?

Reply:
sorry. was confused and pushed to the other repo. Just pushed the change.